### DISCUSSION TITLE
Keine Verbindung zur Fritzbox über WLAN, Ethernet ist ok

### DISCUSSION THREAD

**Discussion by stumpf001** [tbnobody/OpenDTU#437] [Q&A] [answered]
Hallo,

die openDTU funktioniert sehr gut. Erstinbetriebnahme hat einwandfrei funktioniert und ich kann die Daten vom HM-600 auslesen. Allerdings nur, wenn ich mit dem PC/Notebook an der FritzBox über LAN verbunden bin. Wenn ich mit dem Notebook über WLAN die Verbindung zur FritzBox herstelle, kann ich nicht auf die Weboberfläche von der DTU zugreifen. 
Woran kann das liegen?

Danke für die Antwort
Volker

**Comment by Solarteur** [comment]
Hast du versucht, unter WLAN > Funknetz die IP-Adresse deines OpenDTU zu ermitteln? Diese IP-Adresse gebe ich in einen Browser ein, schon erreiche ich die Weboberfläche von OpenDTU. Direkt aus der FritzBox kann ich OpenDTU auch nicht erreichen (PC über WLAN an FritzBox, LAN nicht getestet). Ich weiß auch nicht, ob das überhaupt so vorgesehen ist.

**Comment by Archibald-Harry-Tuttle** [comment]
Wenn Du mit dem Notebook per Wlan an die Fritzbox gehst musst Du die IP Adresse verwenden, welche die Fritzbox per DHCP der OpenDTU vergeben hat.

Also Initialkontakt Notebook zu OpenDTU per Wlan/IP Adresse welche OpenDtu zur Verfügung stellt.
Dies hat funktioniert wie Du schreibst und Du hast sicher dort auch die SSID deiner Fritzbox eingetragen.

Danach meldet sich OpenDTU im WLAN-Netz der Fritzbox an und bekommt von dieser eine neue IP Adresse zugewiesen. Findest Du in der Fritzbox unter Wlan.

**Comment by stumpf001** [comment]
Danke für die Antworten. Die IP ist nicht das Problem, sondern die Verbindung zur FritzBox. Ich kann auf die DTU problemlos über LAN zugreifen, der Vorgang ist ja identisch. Kurz gesagt: LAN Notebook -> FritzBox = OK, aber WLAN Notebook -> FritzBox = Keine Verbindung zur DTU.

**Comment by wib100** [comment]
Kann es sein, dass auf deiner FritzBox eine Art  WLAN-Separation aktiv ist bzw benutzt du das Guest-WLAN?
Kannst du auf andere Geräte im WLAN zugreifen?

**Comment by stumpf001** [comment]
Nein, ich benutze das ganz normale WLAN. Auf andere Geräte kann ich zugreifen und die openDTU taucht in der Netzwerkliste von der Fritzbox auf, ich sehe die zugewiesene IP, aber ich kann nicht über WLAN zugreifen. Über LAN funktioniert das.

**Reply by wib100** [reply]
Geht ein ping auf die IP der OpenDTU?

**Comment by stumpf001** [comment]
Nein, wenn ich vom Notebook ein Ping sende kommt die Antwort: Zielhost nicht erreichbar, stecke ich das LAN ans Notebook, dann kommt der Ping und die Weboberfläche ist ohne Probleme erreichbar.

**Comment by wib100** [comment]
Ok, dann wird es wohl nicht an OpenDTU liegen, sondern eher in deiner Netz-Konfiguration. 
OpenDTU läuft über DHCP. Richtig?
Hast du auf deinem NoteBook am WLAN auch DHCP aktiviert? Schau dir mal die ipconfig des WLAN Adapters an, ob die IPv4 Adresse eh OK aussieht für dein Netz.
Bist du mit dem NB auch am 2.4GHz WLAN?

**Comment by stumpf001** [comment]
ja, die OpenDTU läuft über DHCP, das Notebook auch, allerdings auf 5 GHz. Ist das vieleicht das Problem? Aber das ist ja nur die Verbindung zur Fritzbox. Die Verbindung Fritzbox -> DTU läuft ja einwandfrei.

Protokoll:	Wi-Fi 5 (802.11ac) 
Sicherheitstyp:	WPA2-Personal
Netzfrequenzbereich: 5 GHz
Netzwerkkanal:	36
Verbindungsgeschwindigkeit (Empfang/Übertragung):	234/780 (Mbps)

**Comment by wib100** [comment]
Ev. ist doch eine WLAN Separation im Spiel. Stell mal den NB auch auf 2.4GHz um und teste nochmal.

**Comment by stumpf001** [comment]
Ich hab die Einstellung bei Windows 11 nicht gefunden, wo ich die Frequenz umstellen kann. Aber ich habe mal ein anderes NB probiert. Das läuft mit 2,4GHz, auch hier keine Verbindung über WLAN :(

**Comment by stumpf001** [comment]
Ich denke, der Fehler/Einstellung liegt in der FritzBox, bei allen anderen hier scheint die Verbindung über WLAN ja zu klappen.
Vielen Dank nochmal für die Hilfe.
Gruß Volker

**Comment by wib100** [comment]
D.h. SSID ist für 2.4GHz und 5GHz ist vermutlich gleich. Um ganz sicher zu sein, könntest du das 5GHz kurz für einen Test deaktivieren auf  der FritzBox. 
Leider kenne ich die Möglichkeiten bei der FritzBox zu wenig. Ev. kann da jemand mit einer FritzBox weiter helfen.

**Comment by Solarteur** [comment]
Unter WLAN > Funkkanal > Funkkanal-Einstellungen anpassen
kann ich (FB 7530) das 5-GHz-Frequenzband an- und ausschalten.
Wurde die FB schon einmal neu gestartet - 30 sec von der Spannungsversorgung getrennt?

**Comment by stumpf001** [comment]
Ich hab da alles Versucht, leider ohne Erfolg. Ich werde nochmal suchen und berichten, wenn ich was gefunden habe.

**Comment by MrReSc** [comment]
Du hast zwei verschiedene IP's um OpenDTU via LAN und WLAN zu erreichen, richtig oder?
Jedes Netzwerk Interface bekommt eine eigene IP vom DHCP der FritzBox. Veleicht ist das dein Fehler.

**Reply by stumpf001** [reply]
Nee, ich glaube nicht, dass es an den IPs und DHCP liegt. Folgendes Funktioniert ja, auch mit viel DHCP und IPs:
HM600 -> WLAN -> DTU -> WLAN -> FritzBox -> **LAN** -> NB oder PC
und dies funktioniert nicht:   
HM600 -> WLAN -> DTU -> WLAN -> FritzBox -> **WLAN** -> NB oder PC
Ich glaube, dass liegt allein an den WLAN Einstellungen.

**Comment by koerri** [comment, marked as answer]
Könnte das hiermit zu tun haben?

https://avm.de/service/wissensdatenbank/dok/FRITZ-Box-7590/549_Datei-und-Druckerfreigaben-eines-Computers-sind-im-Heimnetz-nicht-sichtbar/

> K**ommunikation zwischen WLAN-Geräten zulassen**
> Klicken Sie in der [Benutzeroberfläche der FRITZ!Box](https://avm.de/service/wissensdatenbank/dok/FRITZ-Box-7590/1_Benutzeroberflaeche-der-FRITZ-Box-aufrufen/) auf "WLAN".
>     Klicken Sie im Menü "WLAN" auf "Sicherheit".
>     Aktivieren Sie im Abschnitt "Weitere Sicherheitseinstellungen" die Option "Die unten angezeigten aktiven WLAN-Geräte dürfen untereinander kommunizieren".
>     Klicken Sie zum Speichern der Einstellungen auf "Übernehmen".

**Reply by stumpf001** [reply]
Ja, das war die Lösung, jetzt geht es! Die Option "Die unten angezeigten ..." muß angeklickt sein. Danke koerri, und den anderen natürlich auch.

**Reply by stumpf001** [reply]
Und mit Handy geht es jetzt auch, sehr schön.
